models/completion_aggregator/schema.yml

```diff
@@ -0,0 +1,89 @@
+version: 2
+
+models:
+  - name: fact_aggregated_completions
+    database: "{{ env_var('DBT_PROFILE_TARGET_DATABASE', 'reporting') }}"
```

Comment:
I'm seeing the same issue you are and I think this is your problem. Since Clickhouse doesn't have a concept of `database`, only `schema` when `database` and `schema` are both defined dbt_clickhouse seems to treat `database` as a prefix. I think if you delete this line it should work as expected.

FWIW I do also see this successfully being created in a new `reporting_xapi` schema and otherwise it seems to work!

Reply:
Aspects v1.0.2 solved an issue in which the environment variable for the reporting schema was not correctly set.

Reply:
> Since Clickhouse doesn't have a concept of database, only schema when database and schema are both defined dbt_clickhouse seems to treat database as a prefix.

That's exactly what happened! I ended up deleting the `schema="xapi"` instead of removing `database=reporting` here, because the rest of the `fact_*` tables are all under the `reporting` database, cf 8859f6aa0a23381246d7f288b834adc31c6af45e.